src/utils/validators.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ETHUUID } from '@utils';
+import { isValidUuid, isTransactionDataEmpty, isValidAmount } from './validators';
+
+describe('isValidUuid', () => {
+  it('correctly identifies a uuid', () => {
+    const testUuid = ETHUUID;
+    expect(isValidUuid(testUuid)).toBeTruthy();
```

Comment:
When testing a method that returns a boolean, it seems more direct to assert with `toBe(true)` or `toEqual(true)`.
It avoids any ambiguity on the return value of the function.